src/test/java/org/takes/rs/RsXSLTTest.java

```diff
@@ -125,6 +126,46 @@ public Source resolve(final String href,
     }
 
     /**
+     * RsXSLT closes decorated Response body's InputStream when XML conversion
+     * is done.
+     * @throws IOException If some problem inside
+     */
+    @Test
+    public void closesDecoratedResponseInputStream() throws IOException {
+        final String xml = Joiner.on(' ').join(
+            "<?xml-stylesheet href='/b.xsl' type='text/xsl'?>",
+            "<subject>World</subject>"
+        );
+        final String xsl = Joiner.on(' ').join(
+            "<stylesheet xmlns='http://www.w3.org/1999/XSL/Transform'  ",
+            " xmlns:x='http://www.w3.org/1999/xhtml' version='2.0'  >",
+            "<output method='text'/><template match='/'> ",
+            "Hello, <value-of select='/subject'/>!</template></stylesheet>"
+        );
+        final StateAwareInputStream inputStream =
+            new StateAwareInputStream(IOUtils.toInputStream(xml));
+        MatcherAssert.assertThat(
+            new RsPrint(
+                new RsXSLT(
+                    new RsText(inputStream),
+                    new URIResolver() {
+                        @Override
+                        public Source resolve(final String href,
+                                              final String base) {
```

Comment:
@Happy-Neko Too much indentation, should be 4 lines indented from line 153 only.